docs/common-issues.md

```diff
@@ -0,0 +1,101 @@
+# Intro
+
+While reviewing pull requests on `zulip-gci` and submissions in the GCI app, and
+when helping students on Zulip, I've encountered some common problems.
+
+The students are learning a lot and they are eager to tackle the challenges.
+That's why I'm always positive:
+
+  * I thank the student for their contribution
+
+    `Thanks for posting the pull request! Looks great.`
+
+  * I point out the issues in their submission
+
+  * I ask them to fix the issues and thank them
+
+    ```
+    Could you submit your task in the app?
+
+    Thanks!
+    ```
+
+This document is a short list of common issues and some ideas on how to address
+them.
+
+## GitHub
+
+### Student posts a PR without submitting the task
+
+I ask the student if they have claimed and submitted the task:
+
+`This looks great! Have you claimed and submitted the task in the GCI app?`
+
+Some students say they'll do it after their PR is merged - I ask them to submit
+the task in the GCI app, with more details:
+
+```
+You should submit the task in the GCI app, so the mentors know what they should
+review - the flow of reviewing the tasks is first claiming them in the app, then
+submitting and waiting for mentor review, then resubmitting them should there be
+a need for fixes.
+
+Could you submit your task in the app?
+```
+
+### Student opens another PR on the same task
+
+I go to all the previous PRs on this task and close them with comment:
+
+`Another PR for this was opened at <PR_hash>`
+
+I also ask the student to learn how to review and edit their commit history:
+
+```
+You don't have to open a new pull request each time you want to change what
+you've submitted - for future reference you can use
+[this guide](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)
+for fixing your commits.
+```
+
+### Student doesn't know how to edit commit history
+
+I point them to docs:
+
+```
+Some helpful docs:
+
+- [Git guide](http://zulip.readthedocs.io/en/latest/git-guide.html)
+- [working copies](https://github.com/zulip/zulip-gci/blob/master/docs/working-copies.md)
+- [fixing commits](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)
+```
+
+### Student has the same problem for a prolonged time/the problem became complicated
+
+I ask them to join us on Zulip, since it's much easier to discuss there:
+
+`Can you come and chat with us on [Zulip](https://chat.zulip.org/)?`
+
+## GCI app
+
+### Student resubmits the task without properly reviewing it
+
+I ask the student to make sure they don't have any obvious issues in their code
+before resubmitting:
+
+`Please make sure to look through all the files before you submit the task for
+review.`
+
+## Zulip
+
+### Student pings mentors constantly for reviews
+
+I inform the student there is a queue of submissions and their submission will be
+reviewed shortly:
+
+`the task is in the submissions queue, a mentor will get to it at some point
+today`
```

Comment:
I don't know if specifying an exact time is a good idea, since there might be puntual occasions when this may not be achieved (e.g. new year's eve and other festivities). Even though improbable, I think it's better not to compromise.

I'd say "soon", but if you prefer indicating a time, I'd use the 36h established by the GCI rules.